frontend/sw360-portlet/src/main/resources/sw360.properties

```diff
@@ -7,8 +7,21 @@
 # without any warranty.
 #
 
-backend.url= http://127.0.0.1:8080
+# where the frontend searches the backend
+backend.url= http://127.0.0.1:8085
```

Comment:
As you already have mentioned: the URL of tomcat on the current single-tomcat-setup is http://127.0.0.1:8080